1_basics.ipynb

```diff
@@ -82,11 +159,22 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "notebook上にはコマンドを実行した時点の情報が保存されます。例えば以下のコマンドを実行すると時刻が更新されます。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Result at the time of executing the command is saved on the notebook. For example, if you execute the command below, the time will be updated to the current:"
```

Comment:
I produced 4 possible replacements.  Each one deals with the singular/plural problem.  Please feel free to choose the one that most appropriately matches the example on the page:

①"The result at the time the command was executed  is saved in the notebook. For example, if you execute the command below, the time will be updated to the current time:"

②"The result at the time the commands were executed  is saved in the notebook. For example, if you execute the commands below, the time will be updated to the current time:"

③"The results at the time the command was executed  is saved in the notebook. For example, if you execute the command below, the time will be updated to the current time:"

④"The results at the time the commands were executed  is saved in the notebook. For example, if you execute the commands below, the time will be updated to the current time:"

Reply:
Thank you, option (2) seems the best. command=prural. result=single.